src/package.xml

```diff
@@ -754,6 +769,7 @@
         <members>Program_Enrollment__c-HEDA Program Enrollment Layout</members>
         <members>Program_Plan__c-HEDA Program Plan Layout</members>
         <members>Relationship__c-HEDA Relationship Layout</members>
+        <members>Term_Grade__c-Term Grade Layout</members>
```

Comment:
It's actually the HEDA Term Grade Layout (so I don't think this layout would deploy successfully to a new org)

Reply:
It did actually deploy successfully - surprising, but it should still be updated to "HEDA" here.